assets/stylesheets/join.css

```diff
@@ -1,35 +1,245 @@
-  border-radius: 12px;
-  margin:         3px 0 3px 3px;
-  padding:        5px;
+.status-value-container {
+  font-weight: bold;
 }
 
-.session-label:not(.active) {
-  box-shadow: 0 0 8px #47daff;
+.session-row.active {
+  font-weight: bold;
+  background-color: rgba(0, 128, 0, 0.5);
 }
 
-.session-label.active {
-  background-color: #47daff;
+.session-row:hover:not(.active) {
+  background-color: rgba(0, 128, 0, 0.2);
 }
 
-.session-option {
+.row-label {
   display: none;
 }
 
+.left-column {
+  width: 50%;
+  margin-right: 30px;
+}
+
+.session-filter-box {
+  width: 100%;
+  height: 40px;
+  padding-left: 10px;
+  padding-right: 40px;
+  background-image: url('https://freerangestock.com/sample/119161/search-icon-vector.jpg');
+  background-repeat: no-repeat;
+  background-position: right;
+  background-size: contain;
+}
+
+.session-option-container {
+  width: 100%;
+  height: 50vh;
+  margin-top: 15px;
+  overflow: scroll;
+  box-shadow: inset 0px -25px 25px -25px rgba(128, 128, 128, 0.7);
+}
+
+.session-list-boundary {
+  width: 100%;
+  border: 1px solid black;
+  margin: 25px 0 25px 0;
+}
+
+.user-inputs-container {
+  width: 43vw;
+}
+
+.user-inputs-top-row {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 20px;
+}
+
+.user-inputs-bottom-row {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+}
+
+.form-label {
+  font-size: 18px;
+}
+
+.username {
+  display: block;
+  height: 30px;
+  width: 20vw;
+}
+
+.password {
+  display: block;
+  height: 30px;
+  width: 20vw;
+}
+
+.role-select {
+  display: block;
+  height: 30px;
+  width: 25vw;
+ }
+
+.join-button {
+  display: block;
+  margin-bottom: -3px;
+  width: 10vw;
+  height: 40px;
+  border-radius: 7px;
+  font-size: 16px;
+  background-color: rgba(0, 128, 0, 0.5);
+}
+
+.join-button:hover {
+  filter: brightness(0.7);
+}
+
+.join-button:active {
+  filter: brightness(0.5);
+  box-shadow: 3px 3px rgba(0, 0, 0, 0.8);
+}
+
+.right-column {
+  width: 50%;
+}
+
 .session-preview-image {
+  height: 80%;
+  width: 100%;
+  object-fit: cover;
+  border-radius: 20px;
+}
+
+.chat-box-closed {
+  position: fixed;
+  bottom: 0;
+  right: 50px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  cursor: pointer;
+  border-radius: 15px 15px 0 0;
+  width: 20vw;
+  height: 40px;
+  padding-top: 10px;
+  background-color: #aed8f2;
+  color: black;
+  font-size: 18px;
+  font-weight: bold;
+  margin-top: 10px;
+}
+
+.chat-box-closed-header {
+  display: inline-flex;
+}
+
+.chat-box-closed:hover {
+  filter: brightness(0.9);
+}
+
+.chat-box-closed:active {
+  filter: brightness(0.7);
+}
+
+.expand-chat-arrow {
+  margin-left: 30px;
+  margin-top: 3px;
+  width: 0;
+  height: 0;
+  border-left: 10px solid transparent;
+  border-right: 10px solid transparent;
+  border-bottom: 15px solid black;
+}
+
+.chat-box-open {
+  animation: openChatFade ease 0.5s;
+  display: none;
+  position: fixed;
+  bottom: 0;
+  right: 60px;
```

Comment:
When the chat box is closed, you use `right: 50px;`.  The chat box visibly moves to the side (and out of alignment with the preview image) when opened.  We should be consistent about its positioning.